cfme/infrastructure/provider.py

```diff
@@ -308,9 +309,14 @@ def _do_stats_match(self, client, stats_to_match=None, refresh_timer=None, db=Tr
         for stat in stats_to_match:
             try:
                 cfme_stat = getattr(self, stat)(db=db)
-                logger.info(' Matching stat [%s], Host(%s), CFME(%s)' %
-                            (stat, host_stats[stat], cfme_stat))
-                if host_stats[stat] != cfme_stat:
+                success, method, value = tol_check(host_stats[stat],
+                                                   cfme_stat,
+                                                   min_error=0.05,
+                                                   low_val_correction=1)
+                logger.info(' Matching stat [{}], Host({}), CFME({}), '
+                    'with method "{}" and tol={} is {}'.format(stat, host_stats[stat], cfme_stat,
```

Comment:
It doesn't seem right to report on the tolerance and not be sure if we used tolerance matching on the same line (with method ?, tol=?). If the idea is to introduce error tolerance stats matching functions in a generic way, they'll all have some amount of tolerance, so this looks either over- or under-engineered.

Maybe just don't interpolate the method name. :)